### PR TITLE
Allow sites to use local Sass files

### PR DESCRIPTION
**Pull request by alphapapa**
This processes ".scss" files in the "assets/css" directory, allowing
downstream sites to use Sass as well as plain CSS.

### CODE REVIEW

File: doc/content/getstarted.md [hunk 1]
```diff
@@ -72,8 +72,8 @@ Example `config.yaml` files can be seen in [scientific-python.org](https://githu
 
 ### CSS
 
-To customize styles, add CSS files to the `./assets/css/` directory.
-It's recommended to put your customizations in a file named `custom.css`, but other `css` files added there will also be loaded.
+To customize styles, add CSS and Sass files to the `./assets/css/` directory.
+It's recommended to put your customizations in a file named `custom.css`, but other `css` and `scss` files added there will also be loaded.
 
 CSS files are compiled as Hugo templates, i.e. configuration variables from the `config.yaml` file can be accessed as `{{ .Site.Params.VARIABLE }}`.
```
Comment (stefanv) [inline doc/content/getstarted.md:78]: Should the same then hold for SCSS?

**Comment by stefanv** [comment]
This should be good to go now.

File: layouts/partials/css.html [hunk 2]
```diff
@@ -5,15 +5,14 @@
...
-{{- $sass := (slice "theme-css/pst/pydata-sphinx-theme.scss") -}}
 
 <!-- Fonts -->
 {{- $fontName := .Site.Params.font.name | default "Open Sans" -}}
 {{- $fontFace := replace $fontName " " "+" -}}
 {{- $fontSizes := delimit (.Site.Params.font.sizes | default (slice 300 400 600 700)) ";" -}}
 {{- $fontUrl := printf "https://fonts.googleapis.com/css2?family=%s:wght@%s" $fontFace $fontSizes -}}
 
-<link rel="icon" href="{{ "images/favicon.ico" | relURL }}" />
+<link rel="icon" href="{{ " images/favicon.ico" | relURL }}" />
```
Comment (alphapapa) [inline layouts/partials/css.html:15]: Accidental typo, I guess?

File: doc/content/getstarted.md [hunk 3]
```diff
@@ -72,8 +72,8 @@ Example `config.yaml` files can be seen in [scientific-python.org](https://githu
 
 ### CSS
 
-To customize styles, add CSS files to the `./assets/css/` directory.
-It's recommended to put your customizations in a file named `custom.css`, but other `css` files added there will also be loaded.
+To customize styles, add CSS (`.css`) and Sass (`.scss`) files to the `./assets/css/` directory.
+It's recommended to put your customizations in a file named `custom.css`, but other `.css` and `.scss` files added there will also be loaded.
 
 CSS files are compiled as Hugo templates, i.e. configuration variables from the `config.yaml` file can be accessed as `{{ .Site.Params.VARIABLE }}`.
```
Comment (alphapapa) [inline doc/content/getstarted.md:78]: Did we want to mention that SCSS files are also compiled as templates now?

**Review by alphapapa — COMMENTED** [review]
Ready to merge after these, I think.

**Review by stefanv — APPROVED** [review]

**Pull request merged**